actions/conversation-v1/create-counterexample.js

```diff
@@ -64,13 +64,18 @@ module.exports.test = main;
 function getParams(theParams, theServiceName) {
   if (Object.getOwnPropertyNames(theParams).length === 0) {
     return theParams;
-  } else {
```

Comment:
Something like
```js
const params = Object.assign({}, theParams.__bx_creds[theServiceName], theParams);
delete params.__bx_creds;
```

Should work

Reply:
Just tried this, works for me & is much cleaner.  Thanks @germanattanasio